### PR TITLE
Plugin generator CSSify opt-in

### LINKED ISSUE #134: Browserify support should be opt-in
When running, for example:

~~~
donejs add plugin
~~~

I think the browserify support added to `package.json` should be made opt-in via the CLI.

If not, I think there should at least be some documentation or explanation somewhere explaining why it is necessary and added to `package.json` by default, but I'm not sure where that would exist.

### PR DESCRIPTION
**Pull request by leoj3n**
I will look into writing tests tomorrow; this should close #134 if accepted. :fire::fuelpump::boom::fire_engine:

### CODE REVIEW

**Comment by leoj3n** [comment]
OK so looking into the `test` folder I see [`supermodel.test.js`](https://github.com/donejs/generator-donejs/blob/master/test/supermodel.test.js) and [`component.test.js`](https://github.com/donejs/generator-donejs/blob/master/test/component.test.js) have extensive tests but [`plugin.test.js#L44`](https://github.com/donejs/generator-donejs/blob/master/test/plugin.test.js#L44) simply checks for the "correct exist status" of a run with default options.

So, I assume that I don't need to write any tests for this change, as long as the "correct exist status" is still returned, which seems to be the case. Hope I understood that correctly.

**Comment by daffl** [comment]
Indeed, the plugin generator just runs `npm test` and makes sure everything passes, it doesn't test the build or Browserify at the moment (although you might've prompted me to do that now 😉). This PR is definitely 👍 (I'm not sure why code coverage failed).

File: plugin/index.js [hunk 1]
```diff
@@ -85,6 +85,12 @@ module.exports = generators.Base.extend({
         name: 'npmVersion',
         message: 'NPM version used',
         default: version.major
+      }, {
+        type: 'confirm',
+        name: 'cssify',
+        message: 'Include CSSify?',
```
Comment (marshallswain) [inline plugin/index.js]: I think we need either a short description on why they would want CSSify.  Maybe change this to "Include CSSify for Browserify support" or something more descriptive.

**Comment by leoj3n** [comment]
Incorporated @marshallswain's suggestion.

**Pull request closed without merging**